client/src/styles/Home.styles.ts

```diff
@@ -12,7 +12,7 @@ export const fullScreenStyles = {
   }
 };
```

Comment:
do we want the page to look like visit , with the gap below
<img width="938" alt="image" src="https://user-images.githubusercontent.com/99863251/204051272-eb967200-c553-41d5-9a39-3d782c4dc3bd.png">

Currently the home page has the white container extending till end 
<img width="965" alt="image" src="https://user-images.githubusercontent.com/99863251/204051362-434b39fd-53d1-4633-923e-530a7ae841bd.png">

Also the theme is not reflected in the header for /home, i changed to red, as u can see in /visit the header is red

Reply:
Home page extending till the end of page is as per the figma, and personally i think this looks better, looks more anchoring and larger and like a true "home" page.. The theme issue am looking into it

Reply:
Thank you, this was a miss in providing config.theme to the ThemeProvide. Have pushed a fix!

Reply:
This is good find. I didn't change the color and tested it...

Reply:
![image](https://user-images.githubusercontent.com/100614160/204060290-7a4871aa-a8ab-4301-a41a-91ab971b0309.png)
Yes indeed :) . Confirmed with UX that this behaviour is desirable/ expected